x-pack/plugin/src/test/resources/rest-api-spec/test/token/10_basic.yml

```diff
@@ -1,88 +0,0 @@
----
```

Comment:
why is this file deleted?

Reply:
By mistake when splitting out the HLRC parts to a different PR :/